File: PlatziCodingChallenge_Dia36/NumerosAleatorios.js
// Números Aleatorios en JavaScript

function randomNumbers2(dices, faces) {
    let results = new Array()
    for (let d = 0; d < dices; d++) {
        results.push(Math.ceil(Math.random() * faces))
    }
    return results
}

function throwDices(dices) {
    return randomNumbers2(dices, 6)
}

const numeroDados = parseInt(prompt('¿Cuántos dados vas a tirar?'))
alert(`Sacaste: ${throwDices(numeroDados)}`)

File: PlatziCodingChallenge_Dia36/NumerosAleatorios.py
# Números Aleatorios en Python

from random import choice

def randomNumbers2(dices, faces):
    results = []
    for d in range(0, dices):
        results.append(choice(range(1, faces + 1)))
    return results

def throwDices(dices):
    return randomNumbers2(dices, 6)

numeroDados = int(input('¿Cuántos dados vas a tirar?: '))
print('Sacaste: {}'.format(throwDices(numeroDados)))